doc/nrf/ug_radio_coex.rst

```diff
@@ -0,0 +1,196 @@
+
+Enabling CX
+-------------
+
+In order to enable CX:
+
+1. Set the :kconfig:`CONFIG_MPSL_CX` Kconfig option to ``y``.
+2. Enable support for specific Coex implementation. The currently supported implementations are "generic 3-pin interface" and "thread", for
+   details see :ref:`WiFi Coex implementations`.
+#. On nRF53 devices, you must also apply the same Kconfig options mentioned in step 2 to the network core - see :ref:`_ug_multi_image`.
+
+Hardware description
+********************
+
+The |NCS| provides a wrapper that configures WiFi Coexistence based on devicetree (DTS) and Kconfig information.
+
+To enable CX support for any for the currently supported CX implementation, you must add an ``nrf_radio_coex`` node in the devicetree file, as
+described below. The node can also be provided by the devicetree file of the target development kit or by an overlay file. See
+:ref:`zephyr:dt-guide` for more information about the DTS data structure, and :ref:`zephyr:dt_vs_kconfig` for information about differences
+between DTS and Kconfig.
+
+For the currently supported CX implementation the ``nrf_radio_coex`` node has the ``compatible`` property set to
+``generic-radio-coex-three-wire``. Dts node compatible with ``generic-radio-coex-three-wire`` are suitable for a situation in which a 3 wire
+interface to the PTA is provided, ideally such as the one shown in the image below. However, the role of each of the pins is dependent on the Wifi
+Coexistence implementation.
+
+Note: When adding custom user implementation a different name then ``nrf_radio_coex`` can be used for the node. Also, a new device binding can be
+added and used as the ``compatible`` property if ``generic-radio-coex-three-wire`` is unsuitable.
+
+.. figure:: images/coex_generic_3pin_pta.png
+   :alt: Interfacing of a PTA supported by the ``nrf_radio_coex`` node with nRF52
+
+In order to configure hardware for a PTA using a 3 wire interface:
+
+1. Add the following node in the devicetree file:
+
+   .. code-block::
+
+      / {
+            nrf_radio_coex: radio_coex_three_wire {
+               status = "okay";
+               compatible = "generic-radio-coex-three-wire";
+               req-gpios =     <&gpio0 24 (GPIO_ACTIVE_HIGH)>;
+               pri-dir-gpios = <&gpio0 14 (GPIO_ACTIVE_HIGH)>;
+               grant-gpios =   <&gpio0 25 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;
+         };
+      };
+
+#. Optionally replace the node name ``radio_coex_three_wire``.
+#. Replace the pin numbers provided for each of the required properties:
+
+   * ``req-gpios`` - GPIO characteristic of the device that controls the ``REQUEST`` signal of the PTA (request RF medium access).
+   * ``pri-dir-gpios`` - GPIO characteristic of the device that controls the ``PRIORITY`` signal of the PTA (priority + RX/TX direction).
+   * ``grant-gpios`` - GPIO characteristic of the device that controls the ``GRANT`` signal of the PTA (RF medium access granted).
```

Comment:
I would add. ```Note that `GPIO_PULL_UP` is added to avoid floating input pin and is required on some boards only. If target board is designed in such a way that this signal is never left floating, GPIO_PULL_UP can be removed to save power.```